src/type/enum.js

```diff
@@ -0,0 +1,71 @@
+// OpenPGP.js - An OpenPGP implementation in javascript
+// Copyright (C) 2015-2016 Decentral
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 3.0 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+
+/**
+ * Wrapper for enums
+ *
+ * @requires enums
+ * @module type/enum.js
+ */
+
+
+import enums from "../enums";
+
+const type_enum = e => class EnumType {
```

Comment:
This is a bit nitpicky but `e` is a variable name is usually short for `error` - even though I see that apparently enums.js uses `e` as an abbreviation too, but there it refers to the enum _value_, not the enum type. To avoid all confusion I would name it something more explicit here, e.g. `type`?